<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbWidget</class>
 <widget class="QWidget" name="dbWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database</string>
  </property>
  <widget class="QLineEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>280</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>270</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Date</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>831</width>
     <height>201</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>href</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>description</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>type</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>831</width>
     <height>201</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>herf</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>description</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>type</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addallButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Commit All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buildButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>610</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Bulid htm</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>620</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>就绪</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sqlEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>330</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Send SQL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
